macOS/DuckDuckGo/Tab/TabExtensions/TabCrashRecoveryExtension.swift

```diff
@@ -159,7 +159,8 @@ extension TabCrashRecoveryExtension: NavigationResponder {
 
             shouldAutoReload = !isCrashLoop
         } else {
-            shouldAutoReload = featureFlagger.internalUserDecider.isInternalUser
+            // disable auto-reload if tab crash debugging flag is enabled, to allow testing
+            shouldAutoReload = featureFlagger.internalUserDecider.isInternalUser && !featureFlagger.isFeatureOn(.tabCrashDebugging)
```

Comment:
This is the actual change. The rest is about renaming `tabCrashDebugTools` to `tabCrashDebugging` and updating feature flagger mock.